docs/project_wheres_religion_desktop/about.md

```diff
@@ -13,11 +13,12 @@ Where's Religion? is conceptualized and designed for diverse users with interest
 
 - **Source Code:** [https://github.com/oss-slu/lrda_website](https://github.com/oss-slu/lrda_website) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/lrda_website)
 - **Client** Dr. Adam Park
-- **Current Tech Lead:** [Yash Bhatia](https://yashb196.github.io/yashb196/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashb196) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashbhatia238/)
+- **Senior Tech Lead:** [Yash Bhatia](https://yashb196.github.io/yashb196/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashb196) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashbhatia238/)
```

Comment:
The mobile page has @yashb196 listed as "Senior Developer". I don't care which title we use, but we need to be consistent between these projects.

Reply:
issue fixed